<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playerPage</class>
 <widget class="QWidget" name="playerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>138</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>玩家</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>581</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>华文行楷</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>用户名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="username">
          <property name="font">
           <font>
            <family>华文行楷</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>华文行楷</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>经验值：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="expNum">
          <property name="font">
           <font>
            <family>华文行楷</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>华文行楷</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>等级：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="rankNum">
          <property name="font">
           <font>
            <family>华文行楷</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>华文行楷</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>已闯关卡数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="passLevel">
          <property name="font">
           <font>
            <family>华文行楷</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="battleButton">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>对战</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>开始游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
